src/awx_plugins/interfaces/_temporary_private_credential_api.py

```diff
@@ -0,0 +1,52 @@
+# pylint: disable=fixme
```

Comment:
Also, global pylint ignores like this should be used as the last resort. They should be scoped to a single line or fixed instead.